src/components/standard/article.tsx

```diff
@@ -113,9 +116,23 @@ const Standard: FC<Props> = ({ item, children }) => {
 		  )
 		: null;
 
+	const matchScores =
+		'football' in item ? parseMatchScores(item.football) : none;
```

Comment:
@JamieB-gu Thanks for explaining it. I made a change by bringing the parsing into the item.ts. So when we create the MatchReport item, the football property has the Option<MatchScores> type instead of Option<FootballContent>. 

Although it's still a mandatory field in MatchReport. I explained my concern in the https://github.com/guardian/apps-rendering/pull/1173#discussion_r597486499
What do you think about it?